Client/Frontend/Browser/URLBarView.swift

```diff
@@ -141,6 +141,9 @@ class URLBarView: UIView {
 
     fileprivate lazy var scrollToTopButton: UIButton = {
         let button = UIButton()
+        // This button interferes with accessibility of the URL bar as it partially overlays it, and keeps getting the VoiceOver focus instead of the URL bar.
+        // @TODO: figure out if there is an iOS standard way to do this that works with accessibility.
+        button.isAccessibilityElement = false
```

Comment:
This fixes #4992 but it also relates to this bug as it interferes with VoiceOver selection of URL bar buttons.